FoodBoard/Models/FoodBoardContext.cs

```diff
@@ -1,16 +1,21 @@
-﻿using Microsoft.EntityFrameworkCore;
+﻿using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
 
 namespace FoodBoard.Models
 {
-    public class FoodBoardContext : DbContext
+    public class FoodBoardContext : IdentityDbContext<LoginUser>
     {
         public FoodBoardContext(DbContextOptions<FoodBoardContext> options)
             : base(options)
         {
         }
 
         public DbSet<Offer> Offers { get; set; }
-        public DbSet<User> Users { get; set; }
         public DbSet<Groceries> Groceries { get; set; }
```

Comment:
Groceries die wie haben liegen in der DB oder?

Reply:
ja